### PR TITLE
CORE-11842 comment updates

### PR DESCRIPTION
**Pull request by mattbradburyr3**
Factoring in changes to code comments spotted in the update to the docs (mostly RCP -> REST)

### CODE REVIEW

File: workflows/src/main/kotlin/com/r3/developers/csdetemplate/flowexample/workflows/MyFirstFlow.kt [hunk 1]
```diff
@@ -87,9 +87,8 @@ class MyFirstFlow: ClientStartableFlow {
         // Receive a response from the Responder flow
         val response = session.receive(Message::class.java)
 
-        // The return value of a RPCStartableFlow must always be a String, this string will be passed
-        // back as the REST RPC response when the status of the flow is queried on Corda, or as the return
-        // value from the flow when testing using the Simulator
+        // The return value of a ClientStartableFlow must always be a String, this string will be passed
```
Comment (abellagonzalo) [inline workflows/src/main/kotlin/com/r3/developers/csdetemplate/flowexample/workflows/MyFirstFlow.kt]: Minor: using string capitalize and all small case, is it intended?

**Review by abellagonzalo — APPROVED** [review]
LGTM

File: workflows/src/main/kotlin/com/r3/developers/csdetemplate/flowexample/workflows/MyFirstFlow.kt (hunk 1 above)
Comment (mattbradburyr3) [reply workflows/src/main/kotlin/com/r3/developers/csdetemplate/flowexample/workflows/MyFirstFlow.kt]: I'll fix it.

**Review by yarivus — COMMENTED** [review]
thx @mattbradburyr3

**Pull request merged**